src/Raven.Client/Documents/Operations/ETL/Queue/AwsSqsConnectionSettings.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.ETL.Queue;
+
+public sealed class AwsSqsConnectionSettings
```

Comment:
What is the official name of the service. I think it's "Amazon Simple Queue Service". So I think that the naming we use should be "AmazonSqs" not "AwsSqs".

This applies to server side classes as well

Reply:
fixed